packages/components/organisms/f-content-cards/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -0,0 +1,15 @@
+const { getAccessibilityTestResults } = require('../../../../../../../test/utils/axe-helper');
+const HomePromotionCard1 = require('../../../test-utils/component-objects/f-content-cards-home-promotion-card1.component');
+const card = new HomePromotionCard1();
+
+describe('Accessibility tests', () => {
+    beforeEach(() => {
+        card.open();
+        card.waitForComponent();
+    });
+
+    it('a11y - should test f-contentCard component WCAG compliance', () => {
+        // Act
+        const axeResults = getAccessibilityTestResults('f-contentCard');
```

Comment:
Might be worth changing to `const axeResults = getAccessibilityTestResults('f-contentCard-homePromotion-1');`

Reply:
Also, make sure you add another test case for home-promotion-2 :)